### DISCUSSION TITLE
Cannot activate github copilot with GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Nachiten** [community/community#111330] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

My first question obviously has to be, is still Github Coplit free having Github Pro and a GitHub Student Developer Pack?
If the answer is no, then too bad I would have to pay.
If the answer is yes, then continue reading.

1) I already submitted and got my application APPROVED for GitHub Student Developer Pack (yesterday):
![image1](https://github.com/community/community/assets/39410888/7433e6ae-6a7c-4fe4-89f1-1c14016910d1)

2) Already have Github Pro on my account:
![image2](https://github.com/community/community/assets/39410888/e4b82f01-424c-4012-a9b7-6961d3d7faa6)

3) I consulted benefits here: https://education.github.com/pack
![Screenshot 2](https://github.com/community/community/assets/39410888/b5eba2f6-f091-4773-8512-a453d0dc6d19)
And it told me to go to my config inside Copilot

4) In this config, I dont see the message "You're elegible to use copilot free"
![Sceenshot 1](https://github.com/community/community/assets/39410888/65c09b17-e5b0-4a42-b9b5-01433c870ab1)

Why is that it?
I hope I covered everything needed so you have all the info.
Thank you.

**Comment by jasondail** [comment]
I have the same exact issue - application was approved a few days ago.

**Reply by jasondail** [reply]
@Nachiten - just found this - #109705 - might have to wait a few weeks - although for reference my student status has been approved for 6 days now I think.

**Comment by saifeddineelhanoune** [comment]
when you claim the student pack from github copilot implicitly will be authorized from github you just need to install the correct extention in vs code of copilot and sign in with github and congratulation 
you can also download copilot chat and enjoy your journey with coding

**Reply by jasondail** [reply]
I have the extension, I have signed out of github in VS code and browser, signed back in, and synced settings in VS code - and I have authorised github to access VS code. 

I think the main issue is that for me and @Nachiten, we cannot claim the free access for students to Copilot as the directions say in # 3 of @Nachiten's post - so it seems the issue is on the github side, not on the client side.

**Reply by saifeddineelhanoune** [reply]
firstly you should verify that you have claimed the github student pack before installing and doing the process if it is not compeletly claimed you should contact your school to target the issue with the github partnership that's allow what i know brother hope this is helpful

**Comment by Nachiten** [comment, marked as answer]
Solved by myself. Here is how I did it:

1) You need to wait some days for en email to arrive, that marks that the benefits are already available.

2) I went here: https://github.com/features/copilot/plans , make sure you first log in if you are not.

3) Press "Start Free Trial" under "Copilot Individual".

4) It should say "Congratulations, you are elegible for Github Copilot Free". Then you follow the steps and you start a "Subscription" which will be free while you are still a student or meet any other requirement for getting it free. No adding of payment method required.

Thats it, enojoy!!

**Reply by jasondail** [reply]
Thanks very much @Nachiten - this worked for me.

**Reply by 0xWeverton** [reply]
Be careful when doing this step if you are indeed students. A week ago, I went to renew my application and had the status of APPROVED, but even so, I don't have access to GitHub Copilot. I followed the mentioned steps, and now we have this:

![image](https://github.com/user-attachments/assets/71a2f9d2-672e-4833-8d70-5c909eef5f38)
---

If you have an active student plan, there's no reason to start an individual subscription. I tried this way, and now I don't even have access to the page anymore
![image](https://github.com/user-attachments/assets/bbfe1486-f433-47c2-b9f3-0dd156808d27)

----

Remembering that after the mentioned steps, I had the success screen:
![image](https://github.com/user-attachments/assets/5e8485c6-f861-4ce2-8d1e-e8f48d2d686c)

But when trying to log in with the GitHub Copilot extension, I now get the message: [error] [auth] Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as tom-log.

Before attempting this step-by-step, I was receiving the error:  [error] [auth] No access to GitHub Copilot found. You are currently logged in as tom-log

**Reply by Stevenchaosz** [reply]
What helped me was that I realized that I didn't click for access. I thought it would be automatic, since GitHub Copilot is part of GitHub. Anyways, on the [GitHub Student Developer Pack page](https://education.github.com/pack), just click "Get direct access on the GitHub Copilot website."
![CleanShot 2025-04-02 at 22 35 28@2x](https://github.com/user-attachments/assets/e56e04da-9ce3-447c-b54f-4f075e391dca)

**Comment by peterklein2024** [comment]
I faced the same issue with [my site ](https://www.haleproductionstudios.com/)but now it's okay and I resolved it myself by the method above.

**Comment by xxxmoney** [comment]
In my case, I solved it different way
(prerequisities: **Approved** GitHub Student Developer Pack)

Steps:
  1. Go here https://github.com/features/copilot/plans
  2. Click on "Get Started" for Pro tier
  3. Then click on "Get Access To Github Copilot"
  4. You should now see page with confirmation of payment details
  5. Now go back in browser
  6. You should now see a that you have access to Copilot Education

**Comment by yelexin** [comment]
https://education.github.com/pack

![image](https://github.com/user-attachments/assets/ef94c361-ce24-4bb3-aebd-ed8c86d6fee4)

In the offer section, click [Get direct access on the GitHub Copilot website](https://education.github.com/pack/redeem/copilot-student).

**Reply by minhazulislam** [reply]
After many days of trying, this finally worked for me. Thank you so much!

**Reply by Forest-Lee** [reply]
This method works for me to change my account from already registered copilot free to student pack free copilot pro.

**Reply by xiaoran007** [reply]
This method works, thanks

**Reply by tqtss0405** [reply]
yooo this works, thanks so much
